enhancing/enhancer.js

```diff
@@ -5,18 +5,44 @@ module.exports = {
   get,
 };
 
-function succeed(item) {
-  return { ...item };
+function succeed(enhancement) {
+ if(enhancement == 20){
+   return 20
+ }else{
+   return enhancement + 1
+ } 
 }
 
 function fail(item) {
-  return { ...item };
+if(item.enhancement < 15){
+  return {
```

Comment:
While we're only affecting the enhancement and durability values, presumably each item would have other values as well, and we want to make sure we preserve those. Use something like the spread operator or Object.assign() here to make sure the item's other values are included in the return object!